neuralop/layers/spectral_convolution.py

```diff
@@ -416,10 +416,13 @@ def forward(
 
         if self.complex_data:
             x = torch.fft.fftn(x, norm=self.fft_norm, dim=fft_dims)
+            fft_shift_dims = fft_dims
         else: 
             x = torch.fft.rfftn(x, norm=self.fft_norm, dim=fft_dims)
+            fft_shift_dims = fft_dims[:-1] # When x is real in spatial domain, the last half of the last dim is redundant
+        
         if self.order > 1:
```

Comment:
Let's add a short comment to explain - maybe we even add a link to the userguide where we explain it.